config/deployments/production.toml

```diff
@@ -88,55 +93,56 @@ threedsecureio.base_url = "https://service.3dsecure.io"
 netcetera.base_url = "https://{{merchant_endpoint_prefix}}.3ds-server.prev.netcetera-cloud-payment.ch"
 
 [delayed_session_response]
-connectors_with_delayed_session_response = "trustpay,payme"
+connectors_with_delayed_session_response = "trustpay,payme"       # List of connectors which has delayed session response
 
 [dummy_connector]
-assets_base_url = "https://app.hyperswitch.io/assets/TestProcessor/"
-authorize_ttl = 36000
-default_return_url = "https://app.hyperswitch.io/"
-discord_invite_url = "https://discord.gg/wJZ7DVW8mm"
-enabled = false
-payment_complete_duration = 500
-payment_complete_tolerance = 100
-payment_duration = 1000
-payment_retrieve_duration = 500
-payment_retrieve_tolerance = 100
-payment_tolerance = 100
-payment_ttl = 172800
-refund_duration = 1000
-refund_retrieve_duration = 500
-refund_retrieve_tolerance = 100
-refund_tolerance = 100
-refund_ttl = 172800
-slack_invite_url = "https://join.slack.com/t/hyperswitch-io/shared_invite/zt-2awm23agh-p_G5xNpziv6yAiedTkkqLg"
+assets_base_url = "https://app.hyperswitch.io/assets/TestProcessor/"    # Base url for dummy connector assets
+authorize_ttl = 36000                                                   # Time to live for dummy connector authorize request in redis
+default_return_url = "https://app.hyperswitch.io/"                      # Default return url when no return url is passed while payment
+discord_invite_url = "https://discord.gg/wJZ7DVW8mm"                    # Discord invite url for hyperswitch
+enabled = false                                                         # Whether dummy connector is enabled or not
```

Comment:
We can move this to be the first value in this section, so that users easily know that changing this one value is enough for toggling this.